### PR TITLE
Update Stable Version info on package index

### PR DESCRIPTION
**Pull request by joperezr**
cc: @ericstj 

In order to produce this diff, I only ran `dotnet msbuild src\System.Text.Json\pkg\System.Text.Json.pkgproj /t:UpdateRepoPackageIndex /p:UpdateStablePackageInfo=true`. It doesn't have to be System.Text.Json.pkgproj obviously, but you just need to run it on a project that imports the right targets, which would be any *.pkgproj on the repo. Internally, this target is simply updating all of the stable versions based on packages in nuget.org with all versions that are listed.

### CODE REVIEW

File: pkg/Microsoft.Private.PackageBaseline/packageIndex.json [hunk 1]
```diff
@@ -38,6 +38,40 @@
       "InboxOn": {}
     },
     "Microsoft.Build": {
+      "StableVersions": [
```
Comment (ericstj) [inline pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: Technically we don't care about StableVersions for packages we don't build nor depend on in CoreFx.  These are present in the index because they were part of the desktop targeting pack.  Not sure we care enough to remove these, but it is an unnecessary diff that will grow over time.

**Review by ericstj — APPROVED** [review]
I reviewed the diff and it looks OK.  I don't see anything concerning around versions that were missed.  My only real concern here is that some of this is noisy: MSBuild and WCF (and a handful of things with non-official packages).  We could consider removing those from the index, or just ignoring.  @joperezr do you have an opinion?

File: pkg/Microsoft.Private.PackageBaseline/packageIndex.json (hunk 1 above)
Comment (joperezr) [reply pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: Right, I'll remove all of these

**Review by ViktorHofer — APPROVED** [review]
